bedrock/foundation/templates/foundation/index.html

```diff
@@ -26,6 +26,7 @@ <h1 class="page-title">{{ _('The Mozilla Foundation') }}</h1>
   </p>
 
   <p>
+  {% if l10n_has_tag('fxos_removal') %}
```

Comment:
No need for a tag here. None of the /foundation pages are localized (only Annual Report until 2012).